docs/specs/snp-genotyping-vector.md

```diff
@@ -0,0 +1,207 @@
+# SNP genotyping (vector) pipeline specification
+
+* Version: 1.4.0
```

Comment:
Starting out at 1.4.0?

Reply:
Slightly odd I know, but this carries on from the version numbering we had for this spec in the previous repo.